truss/truss_config.py

```diff
@@ -139,6 +143,74 @@ def to_dict(self):
         }
 
 
+@dataclass
+class ExternalDataItem:
+    """A piece of remote data, to be made available to the Truss at serving time.
+
+    Remote data is downloaded and stored under Truss's data directory. Care should be taken
+    to avoid conflicts. This will get precedence if there's overlap.
+    """
+
+    # Url to download the data from.
+    # Currently only files are allowed.
+    url: str
+
+    # This should be relative path. This is where the remote file will be downloaded.
+    at: str
```

Comment:
It's not obvious to me what "at" means, maybe "destination", or "local_destination"?

Reply:
agree, how about local_data_path, to make it clear it's relative to the data directory?